_layouts/styleguide.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
-<html lang="en">
+<!--[if lt IE 9]><html class="lt-ie9"><![endif]-->
```

Comment:
Is this just being used for the styleguide? Are we somehow instructing our users that they need this class to get the library working?